index.html

```diff
@@ -0,0 +1,198 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/39329175e7.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./reset.css">
+    <link rel="stylesheet" href="./style.css">
+    <title>Nova</title>
+</head>
+
+<body>
+    <header class="header-background">
+        <section class="header container">
+            <img class="header__logo" src="./images/logo.svg" alt="nova logo">
+            <nav class="nav">
+                <ul class="nav__items">
+                    <li class="nav__item"> <a class="nav__link" href="#team">The Team</a></li>
+                    <li class="nav__item"> <a class="nav__link" href="#pricing">Pricing</a> </li>
+                    <li class="nav__item"> <a class="nav__link" href="#features">Features</a> </li>
+                </ul>
+            </nav>
+        </section>
+        <section class="hero container">
+            <div class="hero__text-box">
+                <h1 class="hero__heading">Beautiful Free Nova template</h1>
+                <p class="hero__description">A top notch premium quality template for your next web project.</p>
+                <a href="#" class="hero__btn">Download <span>free</span>!</a>
+            </div>
+            <div class="hero__img-box">
+                <img class="hero__img" src="./images/screen.png" alt="hero image">
+            </div>
+        </section>
+    </header>
+
+    <main>
+        <section class="features" id="features">
+            <h2 class="features__header container">Features</h2>
+            <div class="features-box container">
+                <ul class="feature__item">
+                    <li class="feature__icon"><img src="./images/responsive.svg" alt="mobile phone logo"></li>
+                    <li class="feature__name">fully responsive</li>
+                    <li class="feature__description">Looks amazing on any device: smartphone, tablet, laptop and
+                        desktop.</li>
+                </ul>
+                <ul class="feature__item">
+                    <li class="feature__icon"><img src="./images/customizable.svg" alt="tools logo"></li>
+                    <li class="feature__name">customizable</li>
+                    <li class="feature__description">Change the colors, pictures or any of the sections to suit your
+                        needs.</li>
+                </ul>
+                <ul class="feature__item">
+                    <li class="feature__icon"><img src="./images/design.svg" alt="prize medal logo"></li>
+                    <li class="feature__name">slick and beautiful design</li>
+                    <li class="feature__description">Trendy and fresh design, fits any website.</li>
+                </ul>
+            </div>
+
+        </section>
+
+        <section class="featured-in">
```

Comment:
Może `<aside>` jak element dodatkowy/uzupełniający - jeśli go nie będzie to główny przekaz niczym nie traci.